### ISSUE TITLE
Finish writing the Gitter API client

### ISSUE THREAD

**Issue by SQL-MisterMagoo** [Blazored/Gitter#1]
There are stubs for some of the main APis we might need, but many still need adding.

**Comment by SQL-MisterMagoo** [comment]
I don't think we need this as part of RC1 as it already has the parts we need for now.

**Comment by Liques** [comment]
Could be great someday make the Gitter API as an external resource available to the community, maybe via nuget. :-)

**Comment by SQL-MisterMagoo** [comment]
Sure, that's definitely something I'd like to do.

**Comment by chucker** [comment]
@SQL-MisterMagoo regarding @Liques's request, what would you say is the distinction between the Core and Library projects? The Library _almost_ seems suitable to be considered a standalone API client library, but `LocalisationHelper` currently depends on `IJSRuntime`. Can we move `LocalisationHelper` to `Blazor.Gitter.Core` instead? Then `Library` should maybe become a package `Blazored.Gitter.ApiClient`.

**Comment by SQL-MisterMagoo** [comment]
> @SQL-MisterMagoo regarding @Liques's request, what would you say is the distinction between the Core and Library projects? The Library _almost_ seems suitable to be considered a standalone API client library, but `LocalisationHelper` currently depends on `IJSRuntime`. Can we move `LocalisationHelper` to `Blazor.Gitter.Core` instead? Then `Library` should maybe become a package `Blazored.Gitter.ApiClient`.

Sure - that makes sense - Library was intended to be the gitter library - probably localisation ended up there late one night and shouldn't have.

**Comment by chucker** [comment]
>Sure - that makes sense - Library was intended to be the gitter library - probably localisation ended up there late one night and shouldn't have.

Right, thought so. That's #78 then.
